src/contrib/mlir/runtime/cpu/cpu_runtime.hpp

```diff
@@ -38,7 +38,7 @@ namespace ngraph
                 void* allocatedPtr;
                 void* alignedPtr;
                 int64_t offset;
-                int64_t shapeAndStrides[];
+                std::vector<int64_t> shapeAndStrides;
```

Comment:
This representation must match an MLIR representation for the call ABIs to work. 
https://mlir.llvm.org/docs/ConversionToLLVMDialect/

Reply:
Reverted back to what is was before. Suppressing warning on macOS instead.

Reply:
Changed title accordingly